### PR TITLE
spacex group App

### PR DESCRIPTION
**Pull request by businesskaniki**
- [ ] Create app and install basic dependencies 
- [ ]  Fetch missions data
- [ ] Fetch rockets data
- [ ] Display missions - Lists render
- [ ] Display rocket - Lists render
- [ ] Implement rocket booking - Actions
- [ ] Implement mission joining - Actions
- [ ] Implement rocket booking cancelation  and missions
- [ ] Switch badges for Missions and rockets - Conditional components
- [ ] Display reserved rockets and missions in My profile - Filters

### CODE REVIEW

File: src/components/Nav.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        <div className="nav-pills">
+          <NavLink className="link" to="/RocketsList">
+            Rockets
+          </NavLink>
+          <NavLink className="link" to="/Missions">
+            Missions
+          </NavLink>
+          <NavLink className="link" to="/Profile">
+            My profile
+          </NavLink>
```
Comment (Wes-Isaac) [inline src/components/Nav.jsx:25]: - [ ] Great job so far, in the design the button background color and link text color are the same please change the link color.
### Your website

Screen Shot 2022-09-02 at 4 04 07 PM

### Design

Screen Shot 2022-09-02 at 4 04 43 PM

File: package.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    "bootstrap": "^5.2.0",
+    "prop-types": "^15.8.1",
+    "react": "^18.2.0",
+    "react-bootstrap": "^2.5.0",
+    "react-dom": "^18.2.0",
+    "react-redux": "^8.0.2",
+    "react-router": "^6.3.0",
+    "react-router-dom": "^6.3.0",
+    "react-scripts": "5.0.1",
+    "redux-thunk": "^2.4.1",
```
Comment (Wes-Isaac) [inline package.json:18]: - [ ] All the required packages are installed except Redux Logger, please install [Redux Logger](https://www.npmjs.com/package/redux-logger) as specified in the project requirement.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+-react
+-redux
+**To get this project locally.**
+## Run this command on your terminal
+
+
+```
+git clone https://github.com/businesskaniki/spacex.git
+
+```
```
Comment (Wes-Isaac) [inline README.md:18]: - [ ] Kindly add the installation command `npm install`, and how to start the project locally `npm start `step by step.

**Review by Wes-Isaac — CHANGES_REQUESTED** [review]
Hi @businesskaniki & @HatimDev-HE, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- All linters are green ✅.
- Used Gitflow.
- README looks professional 👀.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/components/Nav.jsx (hunk 1 above)
Comment (rloterh) [reply src/components/Nav.jsx:25]: This is well resolved! ✔️

File: README.md (hunk 3 above)
Comment (rloterh) [reply README.md]: This is well resolved! ✔️

File: README.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (rloterh) [inline README.md:65]: - [ ] OPTIONAL
MIT license link is broken, kindly endeavor to fix it, to make your ReadMe appear more professional.

File: README.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+## Authors
+
+👤 **Author1**
+
+-- GitHub: [@githubhandle](https://github.com/businesskaniki)
+- Twitter: [@twitterhandle](https://twitter.com/kaniki7346)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/buinesskaniki)
+
+👤 **Author2**
```
Comment (rloterh) [inline README.md:43]: - [ ] OPTIONAL
Kindly replace the text **Author1** and **Author2**  with your respective full names, so as to make your ReadMe more professional

File: README.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+
+```
+```
+npm install
+
+```
+```
+npm start
+
+```
```
Comment (rloterh) [inline README.md:33]: - [ ] OPTIONAL
It will be good idea to also add instructions on how to run unit test cases for this project.

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Hatim & Nick,  

Your project is complete, great job 🥇! There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Highlights 👍
- Implemented required components
- State management with redux is properly done
- Data is properly fetched from endpoint via store
- Navigations works as intended
- Good unit test coverage with React testing library
- Your app works smoothly
- Reviewed project milestones in turns

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly consider my comments marked **[OPTIONAL]** under this review_
<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**